Assets/MRTK/Services/InputSystem/GazeProvider.cs

```diff
@@ -75,16 +75,15 @@ public class GazeProvider :
         [Tooltip("Maximum head velocity threshold")]
         private float maxHeadVelocityThreshold = 2f;
 
+        #region IMixedRealityGazeProvider Implementation
+
         /// <inheritdoc />
         public bool Enabled
         {
             get { return enabled; }
             set { enabled = value; }
         }
 
-        /// <inheritdoc />
-        public bool IsEyeTrackingEnabled { get; set; }
```

Comment:
This (along with other eye tracking properties that appear removed) isn't removed but moved to the eye gaze implementation region lower down.